common/src/com/thoughtworks/go/domain/Pipeline.java

```diff
@@ -59,7 +59,7 @@ public Pipeline(String pipelineName, BuildCause buildCause, Stage... stages) {
     }
 
     public File defaultWorkingFolder() {
-        return new File(CruiseConfig.WORKING_BASE_DIR + pipelineName);
+        return new File("" + pipelineName);
```

Comment:
Oh, I remember why I did this. It's because it's passed something called a `NullPipeline` in the test (it has `NullStage`s, etc). `NullPipeline` is also used in non-test code (around scheduler) and I couldn't rule out the possibility that it is not ever going to call `defaultWorkingFolder`. I suspect it is not. Do you have any idea if it is, @jyotisingh?

I can change the tests, but don't know if the tests are really representative, in this case.